### PR TITLE
Add config and job for podman 1.6 branch

### PR DESCRIPTION
**Pull request by umohnani8**
This will run the launch-aws openshift test suite
on all PRs in the podman 1.6 branch.

Signed-off-by: Urvashi Mohnani <umohnani@redhat.com>

### CODE REVIEW

**Comment by umohnani8** [comment]
@alvaroaleman hi, any idea why the pj-rehearse job is showing as failed when all the rehearse jobs actually passed?

**Comment by alvaroaleman** [comment]
@umohnani8 Because one of the jobs failed: https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_release/7855/pull-ci-openshift-release-master-pj-rehearse/16920#1:build-log.txt%3A18

It has `skip_report: true` set, that's why you don't see the job itself being reported.

It is a canary of ours, so you can ignore that.

**Comment by umohnani8** [comment]
@alvaroaleman ah got it, thanks! Since everything else is green, this is ready to be merged in then :)

File: ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
+  base-machine:
+    cluster: https://api.ci.openshift.org
```
Comment (hongkailiu) [inline ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: Could you please do the cluster for other base_images too?

Comment (umohnani8) [reply ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: Could you point me to how I can do that?

Comment (stevekuznetsov) [reply ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: He means -- add this `cluster: https...` field to every item in `base_images`

Comment (umohnani8) [reply ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: Ah got it! Fixed :)

File: ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    from: base-machine
+    to: base-machine-with-rpms
+resources:
+  '*':
+    limits: {}
+    requests:
+      cpu: 100m
+      memory: 4Gi
+rpm_build_commands: hack/build-rpms.sh
+tag_specification:
```
Comment (hongkailiu) [inline ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: Need cluster here as well.

Comment (hongkailiu) [reply ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: In general, all isTag reference need cluster for successful run on build01.

See this config as an example.
https://github.com/openshift/release/blob/master/ci-operator/config/openshift/origin/openshift-origin-master.yaml

I will update our doc tomorrow.

Comment (umohnani8) [reply ci-operator/config/containers/libpod/containers-libpod-v1.6.yaml]: Fixed

**Comment by umohnani8** [comment]
This is ready, can we please get it in

**Pull request merged**